middleware/integration_test/docker-compose.yml

```diff
@@ -0,0 +1,135 @@
+version: "3"
+services:
+  bitcoind:
+    image: nicolasdorier/docker-bitcoin:0.17.0
+    networks:
+      local:
+        ipv4_address: 10.10.0.10
+    environment:
```

Comment:
instead of defining all your vars inline with `environment` you can use `env_file` and move all your variables to something like `myenvvariables.env`